index.html

```diff
@@ -173,19 +177,20 @@
       <ol class="algorithm">
         <li>Let <var>cardNetworks</var> be the result of filtering
         <var>data</var>.<a>supportedNetworks</a> for <a>networks</a> that are
-        supported by this payment handler.
+        <a>known</a> by this payment handler.
+        </li>
+        <li>If <var>cardNetworks</var> is empty, set <var>cardNetworks</var> to
+        the payment handler's <a>known networks</a>.
         </li>
         <li>
           <p>
-            Let <var>card</var> be the result of the end user selecting or
-            inputing a <a>card</a> of <a>type</a>
-            <var>data</var>.<a>supportedTypes</a>, or any type if
-            <var>data</var>.<a>supportedTypes</a> is empty, and from one of the
-            <a>networks</a> in <var>cardNetworks</var>, or from any network if
-            <var>cardNetworks</var> is empty. User agents SHOULD validate card
-            details to make sure they adhere to the <a>type</a> and
-            <a>network</a> requirements (e.g., checking that "visa" card starts
-            with the number "4", etc.).
+            Let <var>card</var> be the result of the end user inputting a
+            <a>card</a> of <a>type</a> <var>data</var>.<a>supportedTypes</a>,
+            or any type if <var>data</var>.<a>supportedTypes</a> is empty.
+            Check that <var>card</var> is from one of the <a>networks</a> in
+            <var>cardNetworks</var>. Validate <var>card</var>'s <a>details</a>
```

Comment:
<var>cardNetworks</var>. Validate <var>card</var>'s <a>details</a>

Unless I’m misunderstanding the `Validate … to make sure they adhere to any <a>type</a> and/or <a>network</a> requirements` isn’t not clearly normatively defined and if so is not implementable and should be dropped. Or else this should be replaced with language that is implementable (e.g., using a reference to a normative definition of the *validate* intended here).